src/commands/tcp-redirs.js

```diff
@@ -10,24 +10,45 @@ const Logger = require('../logger.js');
 const application = require('@clevercloud/client/cjs/api/v2/application.js');
 
 async function listNamespaces (params) {
+  const { format } = params.options;
   const namespaces = await Organisation.getNamespaces(params);
 
-  Logger.println('Available namespaces: ' + namespaces.map(({ namespace }) => namespace).join(', '));
-};
+  const formattedNamespaces = namespaces.map(({ namespace }) => namespace);
+
+  switch (format) {
+    case 'json': {
+      Logger.printJson(formattedNamespaces);
+      break;
+    }
+    case 'human':
+    default: {
+      Logger.println('Available namespaces: ' + formattedNamespaces.join(', '));
+    }
+  }
+}
```

Comment:
As namespaces are not always understood well, I suggest to add some information in `human` format for `default`/`cleverapps`

```suggestion
  namespaces.sort((a, b) => a.namespace.localeCompare(b.namespace));

  switch (format) {
    case 'json': {
      Logger.println(JSON.stringify(namespaces, null, 2));
      break;
    }
    case 'human':
    default: {
      Logger.println('Available namespaces:');
      namespaces.forEach(({ namespace }) => {
        switch (namespace) {
          case 'cleverapps':
            Logger.println(`- ${namespace}: for redirections used with 'cleverapps.io' domain`);
            break;
          case 'default':
            Logger.println(`- ${namespace}: for redirections used with custom domains`);
            break;
          default:
            Logger.println(`- ${namespace}`);
        }
      });
      break;
    }
  }
}
```